paasta_tools/cli/fsm/autosuggest.py

```diff
@@ -37,17 +38,16 @@ def _get_smartstack_proxy_port_from_file(root, file):
     return int(port)
 
 
-def suggest_smartstack_proxy_port(yelpsoa_config_root):
-    """Pick the next highest smartstack proxy port from the 20000-21000 block"""
-    max_proxy_port = 0
+def suggest_smartstack_proxy_port(yelpsoa_config_root, range_min=20000, range_max=21000):
+    """Pick a random available port in the 20000-21000 block"""
+    available_proxy_ports = set(range(range_min, range_max + 1))
```

Comment:
No, this would return something in [range_min, range_max].

```
In [1]: range_min = 1

In [2]: range_max = 3

In [3]: range(range_min, range_max + 1)
Out[3]: [1, 2, 3]
```